.github/workflows/builder_docker-based_slsa3.yml

```diff
@@ -323,8 +330,9 @@ jobs:
 
           cat DATA > output-template.json
 
-          jq --argjson subjects "$(<subjects.json)" '.attestations[0].subjects += $subjects' output-template.json > "${GITHUB_WORKSPACE}"/slsa-outputs.json
-          echo "slsa-outputs-name=slsa-outputs.json" >> $GITHUB_OUTPUT
+          jq --argjson subjects "$(<subjects.json)" '.attestations[0].subjects += $subjects' output-template.json > "${GITHUB_WORKSPACE}"/slsa-layout.json
+          echo "slsa-outputs-name=slsa-layout.json" >> $GITHUB_OUTPUT
+          echo "build-outputs-name=build-outputs-${RNG}" >> $GITHUB_OUTPUT
```

Comment:
I saw the note on verifying the hash, lgtm

Reply:
yes! added some more thoughts on that. really, the SLSA layout is what i should rely on